usr/src/test/zfs-tests/tests/functional/inuse/inuse_003_pos.ksh

```diff
@@ -141,6 +141,7 @@ log_note "$UFSDUMP 0bf 512 $rawdisk0 $disk1"
 $UFSDUMP 0bf 512 $rawdisk0 $disk1 &
 PIDUFSDUMP=$!
 
+unset NOINUSE_CHECK
```

Comment:
@jwk404 informed me that I left this out; lets see if this addresses the test failure.